docs/code-search/types/deep-search.mdx

```diff
@@ -2,36 +2,63 @@
 
+For use cases where you're looking for exhaustive answers (e.g. "Find all files with the `.XYZ` file extension in `foo` repo that contain the word `bar`), Code Search still excels, while Deep Search will only utilize a sample of the results. Deep Search will perform a Code Search query as a source, which you can then use to continue with an exhaustive search within the Code Search product.
+
 ### Examples of prompts
 
 - Find examples of logger usage and show examples of the different types of logging we use.
-- I want to know when the indexing queue functionality was last changed in `sourcegraph/zoekt`. Show me the last few commit diffs touching this code and explain the changes.
-- Look at the GraphQL APIs available in `sourcegraph/sourcegraph`. Are any of them unused? The client code is in `sourcegraph/cody`.
+- I want to know when the indexing queue functionality was last changed in `@zoekt`. Show me the last few commit diffs touching this code and explain the changes.
+- Look at the GraphQL APIs available in `@sourcegraph/sourcegraph`. Are any of them unused? The client code is in the `@cody` repository.
 - Which tools do we use in our build processes defined in `BUILD.bazel` files?
 - Generate a request flow diagram for `src/backend`. Mark the auth and rate limit points.
 
-## Enabling Deep Search
+## Conversation sharing
+You can share Deep Search conversations with other users in your Sourcegraph instance. To share a conversation, click the "Share" button in the top left, then copy the link. 
+Once you share a conversation, any user on your instance is able to view it with the link. You can also reset the share link and generate a new one, which invalidates the previous link.
 
-Deep Search can only be used on Sourcegraph instances with Code Search and Cody licenses.
+Note that currently, we do not enforce [permissions](/admin/permissions) for viewing shared Deep Search conversations. This means that any user in your Sourcegraph instance can view any shared conversation, regardless of which repositories they have access to. We plan to add support for permission checking in the future.
 
-Deep Search is disabled by default. To enable it, ask your site administrator to set `experimentalFeatures.deepSearch.enabled = "true"` in your site configuration.
+## Enabling Deep Search
+
+If Deep Search is disabled, ask your site administrator to enable the following setting in your site configuration:
+```json
+"experimentalFeatures": {
+    "deepSearch.enabled": true,
+},
+```
 
 For optimal performance, Deep Search is specialized to only use one model. Currently, Deep Search only supports Claude Sonnet 4.
 
+If you are not using [Cody Gateway](/cody/core-concepts/cody-gateway), you will also need to configure the model through either Amazon Bedrock or GCP Vertex.
+
+### Enabling conversation sharing
```

Comment:
Should we also mention its disabled by default in the section higher up when you mention sharing?